src/otx/api/usecases/exportable_code/demo/requirements.txt

```
@@ -1,4 +1,4 @@
 openvino==2023.3.0
 openvino-model-api==0.1.9
-otx @ git+https://github.com/openvinotoolkit/training_extensions/@74c47db3200000fedb0f0bcf52a3efc22e8dde6c#egg=otx
+otx==1.6.0
```

This will result in error in the exportable code because we have not uploaded 1.6.0 to the public PyPI.

---

Why don't we tag this commit as "rc0" or something and use following dependency?
```suggestion
otx@git+https://github.com/openvinotoolkit/training_extensions.git@1.6.0rc0
```

---

otx @ git+https://github.com/openvinotoolkit/training_extensions/@1.6.0rc0 would also work.